diff --git a/node_modules/worker-loader/dist/workers/InlineWorker.js b/node_modules/worker-loader/dist/workers/InlineWorker.js
index b1e2185..1735299 100644
--- a/node_modules/worker-loader/dist/workers/InlineWorker.js
+++ b/node_modules/worker-loader/dist/workers/InlineWorker.js
@@ -2,7 +2,10 @@
 
 // http://stackoverflow.com/questions/10343913/how-to-create-a-web-worker-from-a-string
 
-var URL = window.URL || window.webkitURL;
+var gl = typeof process !== 'undefined' && Object.prototype.toString.call(process) === 'object process'
+  ? global
+  : window;
+var URL = gl.URL || gl.webkitURL;
 
 module.exports = function (content, url) {
   try {
@@ -11,7 +14,7 @@ module.exports = function (content, url) {
 
       try {
         // BlobBuilder = Deprecated, but widely implemented
-        var BlobBuilder = window.BlobBuilder || window.WebKitBlobBuilder || window.MozBlobBuilder || window.MSBlobBuilder;
+        var BlobBuilder = gl.BlobBuilder || gl.WebKitBlobBuilder || gl.MozBlobBuilder || gl.MSBlobBuilder;
 
         blob = new BlobBuilder();
 
